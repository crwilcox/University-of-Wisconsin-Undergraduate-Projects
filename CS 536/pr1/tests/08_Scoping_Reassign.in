{
= z char
? z
}
{
= z int
? z
}

{
= x int
{
= y double
? x
? y
= x double
? x
}
? y
? x
= x double
? x
}