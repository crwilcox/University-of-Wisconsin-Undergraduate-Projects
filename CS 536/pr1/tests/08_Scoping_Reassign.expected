char
int
int
double
double
undeclared
int
redeclaration
int
